package/MDAnalysis/analysis/rms.py

```diff
@@ -138,70 +137,90 @@
 logger = logging.getLogger('MDAnalysis.analysis.rmsd')
 
 
-def rmsd(a, b, weights=None, center=False):
+def rmsd(a, b, weights=None, center=True, superposition=False):
```

Comment:
If it were completely up to me, I'd make all transformations off by default. However, because we have history here we need to make the current behavior the default.